app/serializers/provider_serializer.rb

```diff
@@ -19,4 +19,44 @@ def institution_type
   def accrediting_provider
     # TODO: pull this thru from UCAS import
   end
+
+  def address1
+    if object.enrichments.present?
+      object.enrichments.last.json_data["Address1"]
+    else
+      object.address1
+    end
```

Comment:
Feels to me like the model should be doing this, not the serializer. i.e. it should decide how to return the correct address info within it's class.

Reply:
Also, @benilovj quite rightly mentioned and I agree that it might make more sense to actually return the address as a hash via a single attribute, `address` or something similar, instead of having each line as a separate attribute at the top of the data structure.